README.md

```diff
@@ -10,6 +10,8 @@ Build/Runs your crystal application, watches files, and rebuilds/reruns app on f
 
 ## Installation
 
+### Installation using ruby script
```

Comment:
@samueleaton Please change if needed, as English is not my native language, I might use some cumbersome words to describe things.